frontend/src/metabase/common/components/FilterPicker/CoordinateFilterPicker/CoordinateFilterPicker.unit.spec.tsx

```diff
@@ -143,11 +143,29 @@ describe("CoordinateFilterPicker", () => {
           expect(getNextFilterColumnNames().column).toBe("User → Latitude");
         },
       );
+
+      it("should add a filter with one value via keyboard", async () => {
+        const { onChange, getNextFilterParts, getNextFilterColumnNames } =
+          setup();
+
+        await setOperator("Less than");
+        const input = screen.getByPlaceholderText("Enter a number");
+        userEvent.type(input, "{enter}");
+        expect(onChange).not.toHaveBeenCalled();
```

Comment:
Here we test that the validation logic works - it's not possible to submit an invalid filter.